.github/workflows/docker-images-nightly.yml

```diff
@@ -64,8 +71,23 @@ jobs:
           [[ "${{github.event.inputs.merge-master}}" == "true" ]] && git remote add upstream https://github.com/n8n-io/n8n.git -f; git merge upstream/master --allow-unrelated-histories || echo ""
         shell: bash
 
-      - name: Build and push
+      - name: Build and push to DockerHub
+        uses: docker/build-push-action@v5.1.0
+        with:
+          context: .
+          file: ./docker/images/n8n-custom/Dockerfile
+          build-args: |
+            N8N_RELEASE_TYPE=nightly
+          platforms: linux/amd64,linux/arm64
```

Comment:
Do we need both for benchmarks?

Reply:
Yes, amd64 for cloud env and arm64 for macs